desktop/lib.go

```diff
@@ -62,14 +62,18 @@ var issueMap = map[string]string{
 }
 
 //export start
-func start() {
+func start(cfgDir, proxyAll *C.char) {
```

Comment:
I don't think this is a sustainable model for passing configuration from the command-line to flashlight...

There are [many flags](https://github.com/getlantern/flashlight/blob/0aa683aeb620fd3d6e318cf788347b7b9c03429f/flags.go#L14-L44) flashlight reads in.  Some of these we may not use often, but looking through the list, I see at least 7 which we use regularly.  This also complicates the process of adding new flags.

However, you can pass arguments to flashlight through environment variables, prefixed with `LANTERN_` like `LANTERN_HEADLESS=true`.  Will that work here?  Will flutter pass environment variables through so that they can be read by `flashlight.ParseFlags`?  If so, `hit_lc_proxy.bash` could define `LANTERN_CONFIGDIR` and `LANTERN_PROXYALL`.  Conveniently, users of the script could define flags as needed via environment variables as well.

I tried to test this myself, but I can't get the script to work.

Reply:
> However, you can pass arguments to flashlight through environment variables, prefixed with LANTERN_ like LANTERN_HEADLESS=true

@hwh33 Yeah, I initially asked on Slack if we wanted to just set these arguments using `.env`:  "Right now, we are using a .env to configure the application so I think that would be the best place to add support for it? So if a CONFIG_DIR is present there, then we  override the config directory used [here](https://github.com/getlantern/lantern-client/blob/main/desktop/lib.go#L647) with it. On the Go side, we could just use https://github.com/joho/godotenv"

Reply:
>  If so, hit_lc_proxy.bash could define LANTERN_CONFIGDIR and LANTERN_PROXYALL

Just made that change to the script, and I confirmed it works.

```bash
LANTERN_CONFIGDIR=$TMPDIR LANTERN_PROXYALL=true flutter run -d macOS
```

Reply:
> Yep, I initially asked on Slack if we wanted to use a .env to pass the args with environment variables.. Let me make that change

Oh sorry for over-explaining then!  I didn't realize that was what you meant.

> Just made that change to the script, and I confirmed it works.

Awesome, thanks!